src/globalStyles.js

```diff
@@ -1,8 +1,9 @@
-import { createGlobalStyle } from 'styled-components';
+import { createGlobalStyle } from "styled-components";
 
 export const GlobalStyle = createGlobalStyle`
   html, body {
     height: 100%;
+    min-width: 400px;
```

Comment:
why `400px`?

`360x640` is still a popular resolution https://gs.statcounter.com/screen-resolution-stats/mobile/worldwide

Why do we need to set minimal width for body at all?